### PR TITLE
Fix tests failing with 3.10.0a2+

### PR DESCRIPTION
**Pull request by noelevans**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@noelevans

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by noelevans** [comment]
I have signed the CLA in the last few minutes. This fixes issue #766.

File: typing_extensions/src_py3/test_typing_extensions.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 OLD_GENERICS = False
 try:
-    from typing import _type_vars, _next_in_mro, _type_check  # noqa
+    from typing import _type_vars, _next_in_mro, _type_check # noqa
```
Comment (gvanrossum) [inline typing_extensions/src_py3/test_typing_extensions.py:38]:
```suggestion
    from typing import _type_vars, _next_in_mro, _type_check  # noqa
```

**Review by gvanrossum — COMMENTED** [review]
Couldn't you simplify the tests by introducing a little helper that evaluates `ForwardRef` instances? Similar to `typing.get_type_hint()` but simpler. (Or maybe just call that?)

File: typing_extensions/src_py3/test_typing_extensions.py (hunk 1 above)
Comment (noelevans) [reply typing_extensions/src_py3/test_typing_extensions.py]: > ... Or maybe just call that

Yes, that makes the change significantly smaller - thanks

**Review by gvanrossum — APPROVED** [review]
Thanks! I'll merge once the CLA comes through. (You have to check occasionally, sometimes it doesn't update automatically. The bot's message shows how.)

**Comment by noelevans** [comment]
It's about 22 hours since I signed the CLA. I assume it has still not gone through yet because of this line in Pull Request Guide:

> After signing the CLA, please **wait at least one US business day** and then check the status by going to the check-python-cla website

https://devguide.python.org/pullrequest/#licensing.

So I assume this will be acknowledged on Monday following Thanksgiving today and Native American Heritage Day tomorrow.

**Comment by noelevans** [comment]
I see the the-nights-who-say-ni have now changed the CLA state to signed. And my account looks correct on bugs.python.org too. Is the PR ready now to be pushed?

**Comment by gvanrossum** [comment]
Thanks for your contribution! Thanks for pinging me, and for your patience with our CLA process over the Thanksgiving holiday.

**Pull request merged**